scripts/update/Updater.php

```diff
@@ -912,5 +912,11 @@ public function update($initialVersion)
         }
 
         $this->skip('17.3.0', '19.3.3');
+
+        if ($this->isVersion('19.3.3')) {
+            $script = 'sudo -u www-data php index.php \'oat\taoProctoring\scripts\tools\KvMonitoringMigration\' -f test_taker_first_name,test_taker_last_name,remaining_time,extended_time,extra_time,consumed_extra_time,last_test_taker_activity -d 1 -s 0 -pc';
```

Comment:
When the data migration script is executed, if the columns are not present in `delivery_monitoring` table then they're created with a default type of `text`.
https://github.com/oat-sa/extension-tao-proctoring/blob/f3934acb851fe1d047fbcdf03a33801a058ac567/scripts/tools/KvMonitoringMigration.php#L206
Columns should be created before the data migration.